apps/web/pages/index.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import Router from 'next/router';
+import { Search } from '@kiwicom/margarita-core';
+
+export default () => (
+  <View>
+    <Search navigation={{ navigate: url => Router.push(`/${url}`) }} />
```

Comment:
it would be nice to have some more re-usable way of injecting this prop. 

`react-navigation` has the `withNavigation` HOC, so we can easily use navigation from anywhere without props-drilling the navigation down to the component that uses it. 

Would it be possible also a component that makes it work also for web?

Like withNavigation.web.js and withNavigation.native.js?

Reply:
Yes, that was my plan eventually. I think I will add this in a different PR. What do you think?

Reply:
Sure, I am ok with that. We should create an issue for it.

Reply:
https://github.com/kiwicom/margarita/issues/50